app/logic/Calendar/Event.ts

```diff
@@ -452,6 +452,11 @@ export class Event extends Observable {
     return true;
   }
 
+  /** Only for incoming meetings */
+  get isCancelled(): boolean {
+    return !this.participants.some(participant => participant.response == InvitationResponse.Organizer);
```

Comment:
Can we please make this cancelled flag more explicit? Capture the info when it reached us.

Reply:
#718